exercises/forth/canonical-data.json

```diff
@@ -452,6 +461,24 @@
             "instructions": ["foo"]
           },
           "expected": {"error": "undefined operation"}
+        },
+        {
+          "uuid": "3c8bfef3-edbb-49c1-9993-21d4030043cb",
+          "description": "only defines locally",
+          "comments": [
+            "This test is meant to determine if instructions that define      ",
+            "words are executed in the state of the program or locally. For   ",
+            "example, in OOP-based approaches, the words should be defined in ",
+            "the same scope, or local to the instance the instructions were   ",
+            "evaluated in.                                                    "
```

Comment:
To me, the padding could be cause for noisy diffs in the future, while sentence per line could make sense in this context as well.